app/templates/components/supplementary-file-browser.hbs

```diff
@@ -3,10 +3,10 @@
 {{/file-renderer}}
 
 <div class="row">
-    <div class="col-sm-8">
+    <div class="col-sm-6">
         <p class="f-w-lg p-b-md">{{selectedFile.name}}</p>
     </div>
-    <div class="col-sm-4 text-nowrap supplemental-downloads">
+    <div class="col-sm-6 text-nowrap supplemental-downloads">
```

Comment:
Just as a small note: if we want to preserve the large-device layout while *also* accommodating smaller screens, I believe bootstrap provides a way to specify different column layouts simultaneously. Eg:
https://getbootstrap.com/examples/grid/

(I'll functionally test this next. Unfortunately docker-sync has gone rogue at the moment)